### DISCUSSION TITLE
What else is being added to the built file?

### DISCUSSION THREAD

**Discussion by Blank-c** [pyinstaller/pyinstaller#7764] [PyInstaller]
When building in ondir mode, I see that the size of the executable was more than 1.5 MB. After stripping the file, it became 254 KB.
The python file was not so large to take up 1 MB of space, to be honest, it contains less than 20 lines without any 3rd party libs.

So my question is, what else is inside this executable?

**Comment by bwoodsend** [comment]
Python's standard library.

**Reply by Blank-c** [reply]
There was already a library zip file which contained the files of stl.

**Reply by bwoodsend** [reply]
Only a subset go into that zip file.

**Reply by Blank-c** [reply]
why not all?

**Reply by bwoodsend** [reply]
`base_library.zip` isn't a PyInstaller specific thing - a regular Python environment likely has one somewhere. It contains only the modules Python requires to get itself sufficiently initialised that the rest of its import machinery can function. I guess there's no reason not to put other modules in `base_library.zip` but then there's no reason to do so either.
